src/index.ts

```diff
@@ -62,7 +62,7 @@ async function onTick() {
     for (const feature of features) {
         if (feature.nextPing.isAfter(moment())) continue;
 
-        await client.sendText(noticeRoomId, `Hey team, ${feature.feature} was last modified ${feature.lastModified.fromNow()} by ${feature.author} - what's the plan?`);
+        await client.sendText(noticeRoomId, `Hey team, ${feature.feature} was last modified ${feature.lastModified.fromNow()} by ${feature.author} - what's the plan? No response necessary. The next ping will be in ${DEFAULT_NOTIFICATION_DAYS} days or you can mute for longer with the \`!feature mute ${feature.feature} 30\` command.`);
```

Comment:
I've done this dance wrong every time:

 - https://matrix.to/#/!bEWtlqtDwCLFIAKAcv:matrix.org/$m7lBy_DdcVgdQpzVNFsi_IVjzQfTDqW7YDPrjn8bpsQ?via=matrix.org&via=element.io&via=vector.modular.im
 - https://matrix.to/#/!bEWtlqtDwCLFIAKAcv:matrix.org/$HIuPmcPyXM8EOfFXuMgEnEy5cgSxcJXN0APGpmilufY?via=matrix.org&via=element.io&via=vector.modular.im
 - https://matrix.to/#/!bEWtlqtDwCLFIAKAcv:matrix.org/$c7zvUHDnEFN0XvljnWtykgOPiSsXOWKKL9COswPtC7E?via=matrix.org&via=element.io&via=vector.modular.im